deploy/device-operator.yaml

```diff
@@ -130,6 +130,7 @@ spec:
                 enum:
                 - Pending
                 - Ready
+                - Failed
```

Comment:
The yaml files are auto generated from the apis defined [here](https://github.com/openebs/device-localpv/tree/develop/pkg/apis/openebs.io/device/v1alpha1). So the changes should be made in the APIs and these files should be generated using `make controller-gen` and `make manifests`. I would suggest to  change [this](https://github.com/openebs/device-localpv/blob/develop/pkg/apis/openebs.io/device/v1alpha1/devicevolume.go#L81) to a typed string and then add `Failed`  as a new state.

Reply:
@vakul-gupta-flp lets make changes at crd comments? Refer to the documentation of [kubebuilder marker](https://book.kubebuilder.io/reference/markers.html) to more undestanding about it.

Reply:
resolved